press/legacy_publishing/module.py

```diff
@@ -112,4 +112,21 @@ def publish_legacy_page(model, metadata, submission, db_conn):
 
     # TODO Insert resource files (images, pdfs, etc.)
 
+    # Copy over existing module_files entries
+    stmt = text(
+        'INSERT INTO module_files '
+        'SELECT :module_ident, fileid, filename '
+        'FROM module_files '
+        'WHERE module_ident = :previous_module_ident '
+        '      AND filename != all(SELECT filename '
+        '                          FROM module_files '
+        '                          WHERE module_ident = :module_ident)'
+    )
+    db_conn.execute(
+        stmt,
+        module_ident=ident,
+        previous_module_ident=existing_module.module_ident,
+        ignore_files=['index.cnxml', 'index.cnxml.html'],
```

Comment:
Would be nice to DRY this between modules and collections. Hmm, the `ignore_files` param seems to be unused?

Reply:
This is what I was using prior to switching to the select that you pointed out above. This will also explain why the `all()` is in use, because it started as analysis of an array. 

I blame making this commit at midnight. 😆